### DISCUSSION TITLE
Which syntax to use in the documentation?

### DISCUSSION THREAD

**Discussion by castelao** [OceanGlidersCommunity/Realtime-QC#4] [Polls]
@soerenthomsen raised the question of which syntax to follow. Is there anyone with strong opinions and technical arguments?

Votes without a technical argument, just an opinion, still count. Of course, a good technical argument wins.

**Comment by soerenthomsen** [comment]
I voted for Jupyter book:
+ easier alignment with other already existing SOPs, which all have a parameter specific RTQC section, if we author in *.md easy to move content back and forth, avoid silo working etc
+ executable, more relevant on the long run i.e. to show tests codes and with test datasets etc. 
+ only markdown (*.md) knowledge necessary to edit existing Jupyter Book

**Comment by soerenthomsen** [comment]
FYI @callumrollo @isgiddy
